.github/sync-files.yaml

```diff
@@ -89,3 +89,4 @@
   files:
     - source: .github/workflows/deploy-docs.yaml
     - source: .github/workflows/delete-closed-pr-docs.yaml
+    - source: mkdocs.yaml
```

Comment:
Like this. (I haven't tested it.)

```suggestion
    - source: mkdocs.yaml
      pre-commands: |
        sd "Autoware Documentation" "Autoware Universe Documentation" {source}
        sd "autoware-documentation" "autoware.universe" {source}
        sd "repo_url: .*" "repo_url: https://github.com/autowarefoundation/autoware.universe" {source}
        sd "docs_dir: .*" "docs_dir: ." {source}
        sd "assets/(.*)" "docs/assets/$1" {source}
```

Reply:
For this part, you should update the `autoware-documentation` side.
![image](https://user-images.githubusercontent.com/31987104/236123678-f0611836-68b1-49a7-8120-6068b6779861.png)

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/361

Reply:
It looks good to me.
https://github.com/autowarefoundation/autoware.universe/pull/3619

Reply:
I'll fix it. :pleading_face: 
![image](https://user-images.githubusercontent.com/44889564/236148786-131232e1-c7c3-4486-b4b0-5b3be93f525b.png)

Reply:
It looks good to me.
https://github.com/autowarefoundation/autoware.universe/pull/3622/files